packages/manager/modules/pci/src/projects/project/translations/Messages_fr_FR.json

```diff
@@ -15,5 +15,6 @@
 
   "pci_projects_project_welcome": "Bienvenue sur la nouvelle interface Public Cloud",
   "pci_projects_project_keystone_upgrade": "Changement important le 23 juin 2020, merci d'anticiper dès maintenant le passage à <strong>Keystone v3</strong>.",
-  "pci_projects_project_keystone_upgrade_more": "En savoir plus"
+  "pci_projects_project_keystone_upgrade_more": "En savoir plus",
```

Comment:
Why is this in English ? 🤔

Reply:
are you refering to "Quota and localization"? it's given like this in functional spec. 

Par mesure de sécurité, votre projet est soumis à un quota. Si vous voulez augmenter ce quota, rendez-vous sur la page Quota and localisation

Reply:
@FredericEspiau It was decided that public cloud sidebar would not be translated